### ISSUE TITLE
Error with version 0.0.4

### ISSUE THREAD

**Issue by mhosman** [zerohouse/router-tab#5]
Error happends with 0.0.4 (from NPM). Not all the folders are present in the package.

`ERROR in src/app/Module/AppModule.ts(68,33): error TS2307: Cannot find module '@zerohouse/router-tab'.`

**Comment by dashkan** [comment]
I see this as well. Happened to two of my packages after upgrading this morning.

Looks like the main and typings entries are missing from package.json file in node_modules/@zerohouse/router-tab. In addition all the compiled output is missing.

Version 0.0.3
```javascript
{
  "_args": [
    [
      "@zerohouse/router-tab@0.0.3",
      "/Users/ashkan/Projects/nbc/dali/src/UI/Dali.UI/Client"
    ]
  ],
  "_from": "@zerohouse/router-tab@0.0.3",
  "_id": "@zerohouse/router-tab@0.0.3",
  "_inBundle": false,
  "_integrity": "sha512-ogNZKSvmbgNaOQsd4+NeAWg0SVDkM7RkT2M2xfjVm/Q7BG0IAtXJkTlrXzD1UL4DCYneR3NSy7UiA+74tUuIpQ==",
  "_location": "/@zerohouse/router-tab",
  "_phantomChildren": {},
  "_requested": {
    "type": "version",
    "registry": true,
    "raw": "@zerohouse/router-tab@0.0.3",
    "name": "@zerohouse/router-tab",
    "escapedName": "@zerohouse%2frouter-tab",
    "scope": "@zerohouse",
    "rawSpec": "0.0.3",
    "saveSpec": null,
    "fetchSpec": "0.0.3"
  },
  "_requiredBy": [
    "/"
  ],
  "_resolved": "https://registry.npmjs.org/@zerohouse/router-tab/-/router-tab-0.0.3.tgz",
  "_spec": "0.0.3",
  "_where": "/Users/ashkan/Projects/nbc/dali/src/UI/Dali.UI/Client",
  "author": {
    "name": "parksungho86@gmail.com"
  },
  "bugs": {
    "url": "https://github.com/zerohouse/router-tab/issues"
  },
  "dependencies": {
    "tslib": "^1.9.0"
  },
  "description": "router-tab directive for mat-tabs",
  "es2015": "fesm2015/zerohouse-router-tab.js",
  "esm2015": "esm2015/zerohouse-router-tab.js",
  "esm5": "esm5/zerohouse-router-tab.js",
  "fesm2015": "fesm2015/zerohouse-router-tab.js",
  "fesm5": "fesm5/zerohouse-router-tab.js",
  "homepage": "https://github.com/zerohouse/router-tab#readme",
  "keywords": [
    "router",
    "material",
    "tabs"
  ],
  "license": "MIT",
  "main": "bundles/zerohouse-router-tab.umd.js",
  "metadata": "zerohouse-router-tab.metadata.json",
  "module": "fesm5/zerohouse-router-tab.js",
  "name": "@zerohouse/router-tab",
  "peerDependencies": {
    "@angular/common": "^6.0.0-rc.0 || ^6.0.0",
    "@angular/core": "^6.0.0-rc.0 || ^6.0.0",
    "@angular/cdk": "^6.3.1",
    "@angular/material": "^6.3.1",
    "@angular/router": "^6.0.0-rc.0 || ^6.0.0"
  },
  "repository": {
    "type": "git",
    "url": "git+ssh://git@github.com/zerohouse/router-tab.git"
  },
  "sideEffects": false,
  "typings": "zerohouse-router-tab.d.ts",
  "version": "0.0.3"
}
```


version 0.0.4
```javascript
{
  "_from": "@zerohouse/router-tab@0.0.4",
  "_id": "@zerohouse/router-tab@0.0.4",
  "_inBundle": false,
  "_integrity": "sha512-hELSbSYxcoZcSSuxgZw2RJ74sYwrQAsTDtSkThOuf19NQQdofrIjkQaygZBhG92ZMLKnW/IW+mACPuxWw0u2QA==",
  "_location": "/@zerohouse/router-tab",
  "_phantomChildren": {},
  "_requested": {
    "type": "version",
    "registry": true,
    "raw": "@zerohouse/router-tab@0.0.4",
    "name": "@zerohouse/router-tab",
    "escapedName": "@zerohouse%2frouter-tab",
    "scope": "@zerohouse",
    "rawSpec": "0.0.4",
    "saveSpec": null,
    "fetchSpec": "0.0.4"
  },
  "_requiredBy": [
    "/"
  ],
  "_resolved": "https://registry.npmjs.org/@zerohouse/router-tab/-/router-tab-0.0.4.tgz",
  "_shasum": "a0662be56bc07562ea7f101a04a84d044ee71b96",
  "_spec": "@zerohouse/router-tab@0.0.4",
  "_where": "/Users/ashkan/Projects/nbc/dali/src/UI/Dali.UI/Client",
  "author": {
    "name": "parksungho86@gmail.com"
  },
  "bugs": {
    "url": "https://github.com/zerohouse/router-tab/issues"
  },
  "bundleDependencies": false,
  "deprecated": false,
  "description": "router-tab directive for mat-tabs",
  "homepage": "https://github.com/zerohouse/router-tab#readme",
  "keywords": [
    "router",
    "material",
    "tabs"
  ],
  "license": "MIT",
  "name": "@zerohouse/router-tab",
  "peerDependencies": {
    "@angular/common": "^7.0.0 || ^6.0.0",
    "@angular/core": "^7.0.0 || ^6.0.0",
    "@angular/cdk": "^7.0.0",
    "@angular/material": "^7.0.0",
    "@angular/router": "^7.0.0 || ^6.0.0"
  },
  "repository": {
    "type": "git",
    "url": "git+ssh://git@github.com/zerohouse/router-tab.git"
  },
  "version": "0.0.4"
}
```

**Comment by mhosman** [comment]
I don't have the permissions to publish a 0.0.5 release in NPM 😀😀😀

**Comment by TdoubleG** [comment]
What's the status of this one?

**Comment by ThePatzen** [comment]
Hi! I really really love your package but same issue here :-(

**Comment by zerohouse** [comment]
sorry this fixed 0.0.5
